runtime/src/accounts_db.rs

```diff
@@ -2865,79 +2864,35 @@ impl AccountsDb {
         assert!(self.storage.get_slot_stores(remove_slot).is_none());
     }
 
-    fn include_owner(cluster_type: &ClusterType, slot: Slot) -> bool {
-        // When devnet was moved to stable release channel, it was done without
-        // hashing account.owner. That's because devnet's slot was lower than
-        // 5_800_000 and the release channel's gating lacked ClusterType at the time...
-        match cluster_type {
-            ClusterType::Devnet => slot >= 5_800_000,
-            _ => true,
-        }
-    }
-
```

Comment:
My interpretation is that `include_owner()` now always returns `true`, so I've removed this function entirely. Let me know if that's not right!